src/main/lrsql/system/lrs.clj

```diff
@@ -74,11 +79,16 @@
         (assoc lrs
                :connection connection
                :authority-fn auth-fn
-               :oidc-authority-fn oidc-auth-fn))))
+               :oidc-authority-fn oidc-auth-fn
+               :reaction-channel (react-init/reaction-channel config)))))
   (stop
     [lrs]
     (log/info "Stopping LRS...")
-    (assoc lrs :connection nil :authority-fn nil))
+    (assoc lrs
+           :connection nil
+           :authority-fn nil
+           :oidc-authority-fn nil
```

Comment:
I'm assuming that this line isn't reaction related, but instead an oversight patch?

Reply:
yep, just a little housekeeping